src/utils.rs

```diff
@@ -219,7 +219,7 @@ pub struct Post {
 
 impl Post {
 	// Fetch posts of a user or subreddit and return a vector of posts and the "after" value
-	pub async fn fetch(path: &str, fallback_title: String, quarantine: bool) -> Result<(Vec<Self>, String), String> {
+	pub async fn fetch(path: &str, quarantine: bool) -> Result<(Vec<Self>, String), String> {
```

Comment:
Removed the `fallback_title` hack for comments. Posts on reddit can never have an empty title, so if the title of a fetched item is empty, we know it's a comment.